caddyhttp/httpserver/plugin.go

```diff
@@ -331,6 +331,7 @@ var directives = []string{
 	"log",
 	"gzip",
 	"errors",
+	"minify", // github.com/hacdias/caddy-minify
 	"ipfilter", // github.com/pyed/ipfilter
```

Comment:
You're gonna find this annoying, but there is a single space that your commit needs in order for the CI tests to pass; run `go fmt` on this file and you'll see that the comment should be lined up with the others. Sorry to be nit-picky!